unit-tests/unit-tests-live.cpp

```diff
@@ -5915,3 +5915,114 @@ TEST_CASE("l500_presets_set_preset", "[live]")
        
     }
 }
+
+TEST_CASE("D55 frame drops", "[live]")
+{
+    // Require at least one device to be plugged in
+    rs2::context ctx;
+
+    if (make_context(SECTION_FROM_TEST_NAME, &ctx))
+    {
+        auto list = ctx.query_devices();
+        REQUIRE(list.size());
+        REQUIRE(list.front().supports(RS2_CAMERA_INFO_PRODUCT_LINE));
+        auto dev = list.front();
+
+        // 424x240
+        // 480x270
+        std::vector<std::pair<int, int>> resolutions = { { 424,240 }, { 480, 270 } };
```

Comment:
Quoted from ticket : 
`happens on color 640x360, 480x270` - adjust accordingly
Additional prerequisites to check:  D455 and USB3